frontend/src/Components/ProviderCard.tsx

```diff
@@ -53,64 +54,50 @@ export default function ProviderCard({
-                                    <div
-                                        className="tooltip"
-                                        data-tip="Manage Users"
-                                    >
-                                        <UserGroupIcon
-                                            className="w-4"
-                                            onClick={() =>
-                                                navigate(
-                                                    `/provider-users/${provider.id}`
-                                                )
-                                            }
-                                        />
-                                    </div>
                                 )}
                             </>
                         ) : (
-                            <div
-                                className="tooltip"
-                                data-tip="Register Provider"
-                            >
-                                <LinkIcon
-                                    className="w-4 "
-                                    onClick={() => oidcClient(provider)}
-                                />
-                            </div>
-                        )}
-                        <div className="tooltip" data-tip="Edit Provider">
-                            <PencilSquareIcon
-                                className="w-4"
-                                onClick={() => openEditProvider(provider)}
-                            />
-                        </div>
-                        <div
-                            className="tooltip"
-                            data-tip="Disable/Archive Provider"
-                        >
-                            <XMarkIcon
-                                className="w-4"
-                                onClick={() => archiveProvider(provider)}
+                            <ULIComponent
+                                dataTip={'Register Provider'}
+                                icon={LinkIcon}
+                                onClick={() => oidcClient(provider)}
                             />
-                        </div>
-                    </>
-                ) : (
-                    <div className="tooltip" data-tip="Enable Provider">
-                        <CheckCircleIcon
-                            className="w-4"
+                        )}
+                        <ULIComponent
+                            dataTip={'Edit Provider'}
+                            icon={PencilSquareIcon}
+                            onClick={() => openEditProvider(provider)}
+                        />
+                        <ULIComponent
+                            data-tip={'Disable/Archive Provider'}
+                            icon={XMarkIcon}
                             onClick={() => archiveProvider(provider)}
                         />
```

Comment:
Would you be able to check on your local if this gives a tooltip for you? On my local it doesn't and I'm not sure why.. it could be an issue only I'm having?

Reply:
![image](https://github.com/user-attachments/assets/82897634-2856-46fb-a559-e9e4d0757530)

Reply:
That is the X on the Canvas card

Reply:
No I mean if you hover over the x does it give a tooltip for you?

Reply:
Great catch!!! data-tip= instead of dataTip=